app/api/email/route.tsx

```diff
@@ -0,0 +1,20 @@
+import loopsClient from "@/lib/loops/client";
+import { NextRequest } from "next/server";
+
+export async function POST(req: NextRequest) {
+  const body = await req.json();
+  const email = body.email;
+
+  try {
+    const resp = await loopsClient.createContact(email);
+    return resp;
+  } catch (error) {
+    console.error(error);
+    const message = error instanceof Error ? error.message : "failed";
+    return Response.json({ message }, { status: 400 });
```

Comment:
Enhanced Error Handling:

Use more specific error messages based on the error type for better debugging. Returning a response with status: 400 and a generic error message may not be helpful if debugging is needed.